sensor_msgs/Image image
---
ObjectSceneGraph[] scene_graphs
bool success
string error_message
